### PR TITLE
Add small GUI niceties

### PR DESCRIPTION
**Pull request by janEbert**
- Split divider is correctly positioned to show the right window
- Double clicking in tree view triggers transition

### CODE REVIEW

File: src/viewer.py [hunk 1]
```diff
@@ -530,6 +530,7 @@ def __init__(self):
         # Create tree view widget
         self.tree = wx.TreeCtrl(nb,-1,style=wx.TR_HAS_BUTTONS)
         self.tree.Bind(wx.EVT_TREE_SEL_CHANGED, self.OnTreeSelectionChanged)
+        self.tree.Bind(wx.EVT_TREE_ITEM_ACTIVATED, self.on_trigger_transition)
```
Comment (rhaschke) [inline src/viewer.py]: Either stick to the existing naming convention (_CamelCase_) or convert all method names to _lower_case_.

Comment (janEbert) [reply src/viewer.py]: That method already exists though, I did not add it. Upper camel case is used less than snake case so if you want, I can rename all existing camel case methods to snake case. I don't think it's a fit for this PR though.

**Comment by janEbert** [comment]
If you wait a bit, I'll also add a file selector for the save button to this PR.

File: src/viewer.py (hunk 1 above)
Comment (rhaschke) [reply src/viewer.py]: Yes, please change to snake_case. I'm fine if you do it in a separate PR.

Comment (janEbert) [reply src/viewer.py]: After some thought, due to wxPython having – as you mentioned – upper camel case as its naming convention, maybe we should convert everything to upper camel case. On the other hand, if all other methods are snake case, wx methods stand out nicely.

File: src/viewer.py [hunk 2]
```diff
@@ -654,6 +656,7 @@ def on_set_initial_state(self, event):
 
     def on_trigger_transition(self, event):
         """Event: Change the current state of the server."""
+        # TODO This never works the first time it is executed.
```
Comment (rhaschke) [inline src/viewer.py]: Hi @janEbert,
a possible reason for this issue might be the creation of the transition_pub publisher below.
- First, it is created from scratch for each call, which is not necessary
- Second, the initial creation overhead might cause the issues
Please move the creation of this publisher into the constructor.

**Pull request merged**